libs/payments/paypal/src/lib/paypalCustomer/paypalCustomer.repository.ts

```diff
@@ -152,3 +152,20 @@ export async function deletePaypalCustomer(
     ).numDeletedRows === BigInt(1)
   );
 }
+
+/**
+ * Delete paypalCustomers from the database by uid.
+ *
+ * @returns Number of deleted paypalCustomers
+ */
+export async function deletePaypalCustomersByUid(
+  db: AccountDatabase,
+  uid: Buffer
+): Promise<bigint> {
+  return (
+    await db
+      .deleteFrom('paypalCustomers')
+      .where('uid', '=', uid)
+      .executeTakeFirstOrThrow()
+  ).numDeletedRows;
+}
```

Comment:
Added so that we can clean up paypalCustomer entries. I believe we may need to do some more thinking here, but this is here for now.